docs/guides/opensuse.md

```
@@ -0,0 +1,403 @@
+
+Unlike Fedora, zram is not enabled by default. The `zram-generator` package can be installed to set up and use zram instead of traditional swap. The Arch wiki's [guide on zram](https://wiki.archlinux.org/title/Zram) can be used as a reference. The steps below set up zram using the Fedora 37 default configuration and one additional option:
+
+1. Ensure zswap is disabled to avoid conflicts with zram. The process can be found on the [Arch wiki](https://wiki.archlinux.org/title/zswap#Toggling_zswap).
+
+2. Install **zram-generator** to handle zram creation:
+
+```bash
+sudo zypper in zram-generator
+```
+
+3. Create a zram configuration file at one of **zram-generator**'s expected locations, such as `/usr/lib/systemd/zram-generator.conf`.
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/2/3

(MD029, ol-prefix)